### ISSUE TITLE
Problem with current schematics 

### ISSUE THREAD

**Issue by MoonCutter2B** [Millenaire/Millenaire#4]
(locked)chests are placed as rice
signs are placed as maize
and all torches placed on north and west side of a block drop to the floor.
(south and east are ok)

**Comment by FireController1847** [comment]
Mind describing the issue a little more using the new issue template found [here](https://github.com/Millenaire/Millenaire/blob/master/.github/ISSUE_TEMPLATE.md)?

**Comment by MoonCutter2B** [comment]
Please describe the issue you are having is as much detail as possible:

In creative mod I used the wand of summoning on a sign with name of a building and a level. This was after I fixed issue #3.
After the building has spawned I checked it out. The blocks that think should be mill locked chests was actually planted rice and the the mill signs was actually maize (that had dropped to the ground). I guess the mapping of blocks in the current schematics to the actual blocks are incorrect (or just not up to date).

And some of the torches was laying on the floor instead of placed on the walls. Looks like some kind of direction problem.

Describe how to be able to reproduce this bug:

Further details:

Mod Version: 7.0.0 as found in git this morning. (CET) + my own fixes.
Other Mod Versions (Seperated By Commas): none
Forge Version: latest for 1.8.9

**Comment by FireController1847** [comment]
@MoonCutter2B Please fix your mod version and forge version. "Latest for 1.8.9" and "7.0.0" do not tell us the exact version you are using, as they will change over time (and there are 3 7.0.0 alphas)

**Comment by SamboyCoding** [comment]
Currently fixing this. ETA of wednesday at the latest hopefully.

**Comment by SamboyCoding** [comment]
I've fixed it for the Temple, just need to fix the other buildings.

**Comment by SamboyCoding** [comment]
Once #17 is merged this should be auto-closed. Can confirm it's 100% working now.
